Directory.Build.props

```diff
@@ -9,7 +9,7 @@
 
   <PropertyGroup>
     <Description>A client library for KMD Logic Audit</Description>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>1.4.0</VersionPrefix>
```

Comment:
I would bump version to `1.3.2`

Reply:
Also update `CHANGES.md` document

Reply:
Ok, I just saw @avin101 comment and in the end I agree with you guys. So keep `1.4.0` version, but please update document.

Reply:
yes, will update the document.